### ISSUE TITLE
color contrast error - violation or manual check?

### ISSUE THREAD

**Issue by clblew** [opena11y/fae-ainspector#17]
FAE version 0.9.6b
AIS version 0.16.8.1

FAE notes color contrast as a manual check and AIS notes it as an error. I know there has been a lot of discussion on this. Which is the correct one?

DRES website: http://fae20.cita.illinois.edu/summary/14c2308a68d0be47/rc/8/
Styling/Content, Color 1 	Text must exceed CCR of 4.5 	MC

AIS on http://disability.illinois.edu/athletics/womens-wheelchair-basketball
Text must exceed CCR of 4.5    V    1.4.3   AA   Y
